src/SoapMessageEventSubscriber.php

```diff
@@ -56,9 +56,9 @@ public function addMessage(PreSerializeEvent $event)
                 throw new RuntimeException('Missing XmlRootName or XmlRootNamespace for ' . $event->getType()['name']);
             }
 
-            if($metadata->xmlRootPrefix !== null){
+            if ($metadata->xmlRootPrefix !== null) {
                 $tagName = $metadata->xmlRootPrefix . ':' . $metadata->xmlRootName;
-            }else{
+            } else {
```

Comment:
this else can be removed by moving the assignment before the if. (see SoapHeaderEventSubscriber.php)